gadgets/traceloop/README.mdx

```diff
@@ -146,6 +186,133 @@ Finally, clean the system:
     </TabItem>
 </Tabs>
 
+## Advanced Tracing Scenarios
+
+### Built-in System Call Filtering
+
+For more efficient filtering, traceloop supports built-in syscall filtering at the eBPF level, which reduces overhead and provides cleaner output than post-processing with grep.
```

Comment:
I would avoid mention of grep. We also have built-in filtering with `--filter` or `--filter-expr` done in userspace. See:
https://inspektor-gadget.io/docs/v0.43.0/reference/run#filtering-in-user-space

You can explain that using `--syscall-filters` is more efficient because it is done in eBPF.

Reply:
Alright, will update